code/game/objects/items/storage/belt.dm

```diff
@@ -863,31 +892,18 @@
 	new /obj/item/melee/sabre(src)
 	update_appearance()
 
-/obj/item/storage/belt/plant
-	name = "botanical belt"
-	desc = "A sturdy leather belt used to hold most hydroponics supplies."
-	icon_state = "plantbelt"
-	inhand_icon_state = "utility"
-	worn_icon_state = "plantbelt"
-	content_overlays = TRUE
+/obj/item/storage/belt/sabre/centcom
+	name = "Commander's sabre sheath"
+	desc = "A expensive sheath made of pure gold, and exotic fabric to house an even more expensive sword, this one has CENTRAL COMMAND etched into the gold stripe, and a nice white stripe."
+	icon = 'modular_zubbers/icons/obj/clothing/belts.dmi'
+	icon_state = "cent_sheath"
+	inhand_icon_state = "cent_sheath"
+	worn_icon = 'modular_zubbers/icons/mob/clothing/belts.dmi'
+	worn_icon_state = "cent_sheath"
+	lefthand_file = 'modular_zubbers/icons/mob/inhands/equipment/belt_lefthand.dmi'
+	righthand_file = 'modular_zubbers/icons/mob/inhands/equipment/belt_righthand.dmi'
+
+/obj/item/storage/belt/sabre/centcom/PopulateContents()
+	new /obj/item/melee/sabre/centcom(src)
+	update_appearance()
```

Comment:
Modularize this too and restore this files contents back to normal.

Reply:
I've done that, but apologies.. But, what do you mean by Modularize? You mean put it in the Zubbers folder?